### ISSUE TITLE
[Issue]Connecting to external display makes the Network alert go blank.

### ISSUE THREAD

**Issue by dhairyasd** [henrypp/simplewall#755]
I feel this is an important bug that needs to be addressed urgently. 
Whenever I connect to an external display, simplewall alerts don't display anything. They are just blank windows. The only way to dismiss these window is to press the cross button. Here is a preview->
![simplewall network alert](https://user-images.githubusercontent.com/54510612/90303932-882c3480-de80-11ea-90de-f2e9a7ebf76e.png)
This issue persists even if I disconnect the external display and start reusing the laptop inbuilt display.

Another issue that I faced is that the icons of the main page automatically enlarge after connecting to the external display. Here is a preview->
![simplewall icons](https://user-images.githubusercontent.com/54510612/90303993-37690b80-de81-11ea-871f-cc4f3305fddf.png)

Another thing that I wanted to add, is that, it would be really helpful if you add a feature that lets us know if a blocked application is attempting to connect to the internet on the main window, like, using a purple colored bar to highlight it's attempts to connect to the internet.

Lastly, I am not sure which exact windows application lets windows know that it's connected to the internet, I assume some of the default-blocked system applications attempts to connect to the internet when the Wi-Fi is connected. There are some applications(example Spotify) that first uses that information and then attempt to connect to the internet. Even though I manually "allow" the application to access internet, it asks windows if internet is connected and then starts working. While Simplewall continues to display the purple bar, Spotify continues to display that it's offline. At first, I thought it was an issue on the Spotify application , but later, I found out when I noticed that the Wi-Fi icon displays "Not Connected" when simplewall is switched on. Only way for windows to figure out that it has internet connection is to, first, switch-off Simplewall filters, then switch off/on Wi-Fi and then wait for the Wi-Fi connection to show "Connected".(Then re-enable the simplewall filters) Doing this process maintains the "Connected" status and Spotify starts working. I assume some inbuilt windows system application maintains the internet connection status, and is probably blocked by the simplewall filters. Is it a security issue if we allow this application to run or is it safe?. I am not quite sure, I would like you guys to figure this out, but, it would be really useful if simplewall can bypass this application to let windows know that it has internet connection. 

Thanks for the wonderful application guys, keep up the good work.

**Comment by henrypp** [comment]
check it

**Comment by juandoska** [comment]
> I feel this is an important bug that needs to be addressed urgently. Whenever I connect to an external display, simplewall alerts don't display anything. They are just blank windows. The only way to dismiss these window is to press the cross button. Here is a preview-> ![simplewall network alert](https://user-images.githubusercontent.com/54510612/90303932-882c3480-de80-11ea-90de-f2e9a7ebf76e.png) This issue persists even if I disconnect the external display and start reusing the laptop inbuilt display.
I did have the same issue, the reinstallation of the program solved it. The alerts are displayed everytime a new connection happens
